### ISSUE TITLE
Multiple- SSL, Syntax Error in Load file

### ISSUE THREAD

**Issue by adamphetamine** [dimitri/pgloader#898]
Hi, my pgloader load file is like this-

```
LOAD DATABASE
     FROM mysql://root:/password@172.16.1.199:3306/whd?useSSL=false
     INTO postgresql://postgres:password@172.16.1.209:5432/whd?useSSL=false
     WITH include drop, create tables, no truncate, create indexes, reset sequences, foreign keys, downcase identifiers
     SET maintenance_work_mem to '256MB', work_mem to '48MB', search_path to 'whd'
     BEFORE LOAD DO $$ create schema if not exists whd; $$
     CAST type decimal to numeric
;
```

Results-

1. With SSL off it complains about shared object 'libssl.so.1.0.1e' being missing but appears to proceed if you select '0 Skip this shared object and continue' 
2. with SSL not specified it complains that 'The alien function CRYPTO_num_locks' is undefined' when it is trying to log into the mysql database at 172.16.1.199
3. Installing an up to date version of OpenSSL did not appear to help, and linking the shared object failed as I couldn't figure out the correct path
4. complains about a syntax error near 'not' in 'BEFORE LOAD DO $$ create schema if not exists whd; $$'
5. Bunch of errors after 'Kaboom' just like this one-
https://github.com/dimitri/pgloader/issues/892

*I'm not putting all these errors up because I'm cranky. I'm very grateful for the tool, but I did want to demonstrate that I've spent a lot of time on this before creating this report

.
 
  - [ ] pgloader --version

 ```
  3.4.1 
Compiled with SBCL 1.3.11-1.el7

Running on CentOS 7 with PostgreSQL 9.2
 ```
  
  - [ ] did you test a fresh compile from the source tree?
no
    
    Compiling pgloader from sources is documented in the
    [README](https://github.com/dimitri/pgloader#build-from-sources), it's
    easy to do, and if patches are to be made to fix your bug, you're going
    to have to build from sources to get the fix anyway…
cool, might take me a couple of days as it's after 1am tho
    
  - [ ] did you search for other similar issues?
yes
    
  - [ ] how can I reproduce the bug?
Not sure, I seem to have all of them!

I'll keep working on it, but please let me know if you have any suggestions

**Comment by dimitri** [comment]
Hi,

We have had multiple issues with OpenSSL for quite a while in pgloader development. The way importing shared objects symbols work in Common Lisp is not easy to make compatible with the way openssl is built. I believe the current version of pgloader to be much better equipped in that area. Please try building the latest release from the bundle at https://github.com/dimitri/pgloader/releases/tag/v3.6.1.

**Comment by adamphetamine** [comment]
Thank you. If you see any errors in the load file please let me know as well. I'm going cross eyed!

**Comment by adamphetamine** [comment]
I gave up trying to install pgloader on CentOS 7, but here are a few things that I think could help-

1. I might be blind but the installation docs don't seem to mention running the bootstrap script before running the makefile. I admit this might make me look dumb as Unix isn't my first language, but this is simply feedback...
2. Make seems to crash - I got a crash at 'pgloader.user-symbols' and another at ‘building closure with character runes’ but I guess this could be because I had tried to install the dependencies manually before I discovered the bootstrapper script.
3. I did get the Docker running really easily, then I had to delete the SSL references in my .load file to get it to run. I now have a new error to investigate.

Thanks again for the tool, I think I'm getting close...
